source/systems/research.md

```diff
@@ -17,15 +17,26 @@ Each column of a research tree is also the level the University needs to be to b
 | 4                    | 4                        |
 | 5 and higher         | 5                        |
 
-You can only choose one column 6 research in each of the Combat, Civilian, and Technology trees.
+You can only have one column 6 research in each of the Combat, Civilian, and Technology trees. To unlock a different column 6 research for that tree, you must undo the completed one first.
+
+| Symbol | Description |
+| ------ | ----------- |
+| <img src="../../assets/images/gui/university/university_research_blocked.png" alt="Blocked Research"> | A research can be **blocked**, either by an unfinished prerequisite research, by a completed blocking research, or because the research tree has another column 6 research already active. |
+| <img src="../../assets/images/gui/university/university_research_locked.png" alt="Locked Research"> | A **locked** research requires a building or buildings, or other unrelated research, before it can be started. | 
+| <img src="../../assets/images/gui/university/university_research_unlocked.png" alt="Unlocked Research"> | An **unlocked** research has all colony and research requirements met, but requires an item or items. **These items must be in the player's inventory.** | 
+| <img src="../../assets/images/gui/university/university_research_ready.png" alt="Available Research"> | An **available** research is ready to begin. Clicking the title of the research will consume the items from the player's inventory and start the research. |  
+| <img src="../../assets/images/gui/university/university_research_progress.png" alt="Progressing Research"> | **Progressing** research is your Researcher's current focus. This research will show its current progression, and a rough estimate of remaining time to completion. Progressing research can be canceled by clicking the research title and then clicking the Cancel pop up. Cancelling research will **not** refund material costs. | 
+| <img src="../../assets/images/gui/university/university_research_complete.png" alt="Complete Research"> | **Complete** research has been fully unlocked by your University, and automatically applies its effects to the colony and colonists. | 
+| <img src="../../assets/images/gui/university/university_research_or.png" alt="Exclusive Research"> | Some researches are **Exclusive**, requiring such extreme focus that they aren't compatible with each other. Only one Research from a specific **or** selection may be learned in a colony at a time. | 
+| <img src="../../assets/images/gui/university/university_research_undo.png" alt="Undo Research"> | Completed exclusive researches, or research that depend on exclusives, or tier 6 researches, may be **Undone**. Undoing research does *not* refund the research costs, and consumes the displayed item. | 
```

Comment:
```suggestion
| <img src="../../assets/images/gui/university/university_research_undo.png" alt="Undo Research"> | Completed exclusive researches, researches that depend on exclusives, or tier 6 researches, may be **undone**. Undoing a research does *not* refund the research costs and consumes the displayed item. | 
```
Not all researches can be undone?

Reply:
At the least for now. There's likely always going to be some limitations, because the devs want it so only research without started descendants can be undone, but this one's mostly a tradeoff between limiting the system too much and people accidentally undoing research.  So far, the only excluded case people have asked about is the Night Owl research line, and that honestly seems like it's a better compromise to be 'stuck' lighting your colony up better.

The +27 inventory slot research also can't be undone, even though it'd otherwise be eligible.  I'm not sure what the right behavior for a shrinking inventory should be, but whatever it is, it's not implemented.

Reply:
maybe say what *can't* be undone then, imo it's a little confusing atm